src/open_inwoner/js/components/map/index.js

```diff
@@ -85,7 +86,7 @@ class Map {
     return `
         <div class="leaflet-content-name">
           <h4 class="h4">
-            ${displayName}
+            <a href=${locationDetailView}>${displayName}</a>
```

Comment:
`locationDetailView` is optional/conditional a few lines above and in the model methods, but treated as required here. 

(also I miss the quotes in the href)